tools/accuracy_checker/accuracy_checker/annotation_converters/ms_coco.py

```diff
@@ -262,6 +262,20 @@ class MSCocoSegmentationConverter(MSCocoDetectionConverter):
     __provider__ = 'mscoco_segmentation'
     annotation_types = (CoCoInstanceSegmentationAnnotation, )
 
+    @classmethod
+    def parameters(cls):
+        configuration_parameters = super().parameters()
+        configuration_parameters.update({
+            'semantic_only': BoolField(
```

Comment:
You should update description in `tools/accuracy_checker/accuracy_checker/annotation_converters/README.md`

Reply:
Fixed